### DISCUSSION TITLE
getfrp.sh炸了

### DISCUSSION THREAD

**Discussion by HPPC114514** [natfrp/wiki#176] [使用问题]
如题，cloudflare显示523错误
图：http://ys-o.ysepan.com/618134111/218016806/kjtsh2u46451487I3OIc3/B6888391-2719-49EF-B840-17C1030E90FB.png

_真是让人太开心力（喜（悲（恼_

**Comment by tidymoon** [comment]
emmm这玩意好像被抛弃了？ 具体不清楚 不过它已经从文档移除了
